components/Tabs2/src/MDCTabBarView.m

```diff
@@ -16,4 +16,49 @@
 
 @implementation MDCTabBarView
 
+#pragma mark - Initialization
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    self.items = @[];
```

Comment:
Please assign to the instance variable directly here. Since `self` is not yet fully initialized, the `setItems:` method may fail.

```
if (self) {
  _items = @[];
}
```

Reply:
Thanks fro reminding!